cli/package.json

```diff
@@ -23,11 +23,18 @@
     "@coral-xyz/anchor": "0.27.0",
     "@lightprotocol/zk.js": "0.3.2-alpha.8",
     "@solana/web3.js": "1.76.0",
+    "@types/chai": "^4.3.5",
+    "@types/chai-as-promised": "^7.1.5",
+    "@types/mocha": "^10.0.1",
+    "chai": "^4.3.7",
+    "chai-as-promised": "^7.1.1",
```

Comment:
in other  yarn projects these dependencies are in the dev dependencies.
Do we really need those in dependencies?

Reply:
no need, removed ✅